nodejs/awsx/apigateway/api.ts

```diff
@@ -299,11 +299,13 @@ export interface APIArgs {
 }
 
 export class API extends pulumi.ComponentResource {
-    public restAPI: aws.apigateway.RestApi;
-    public deployment: aws.apigateway.Deployment;
-    public stage: aws.apigateway.Stage;
+    public readonly restAPI: aws.apigateway.RestApi;
+    public readonly deployment: aws.apigateway.Deployment;
+    public readonly stage: aws.apigateway.Stage;
 
-    public url: pulumi.Output<string>;
+    public readonly url: pulumi.Output<string>;
+
+    private readonly swaggerLambdas: SwaggerLambdas;
```

Comment:
Do want to do the same for authorizers? Maybe thats an add on down the line.

Reply:
in general, yes :) Basically any resource we create on their behalf should be reachable somehow.  But i'm willing to wait o it :)